src/catalog/src/information_schema.rs

```diff
@@ -32,14 +36,38 @@ use table::metadata::{
 };
 use table::thin_table::{ThinTable, ThinTableAdapter};
 use table::TableRef;
+pub use table_names::*;
 
 use self::columns::InformationSchemaColumns;
 use crate::error::Result;
+use crate::information_schema::memory_table::{get_schema_columns, MemoryTable};
 use crate::information_schema::tables::InformationSchemaTables;
 use crate::CatalogManager;
 
-pub const TABLES: &str = "tables";
-pub const COLUMNS: &str = "columns";
+lazy_static! {
+    // Memory tables in `information_schema`.
+    static ref MEMORY_TABLES: Vec<&'static str> = vec![
+        ENGINES,
+        COLUMN_PRIVILEGES,
+        COLUMN_STATISTICS
+    ];
+}
```

Comment:
```suggestion
/// Memory tables in `information_schema`.
const MEMORY_TABLES: &[&str; 3] = &[ENGINES, COLUMN_PRIVILEGES, COLUMN_STATISTICS];
```

Reply:
We will continue adding tables, it's cumbersome to modify the array length every time.

Reply:
Ah, the length is not required. Try this:

```suggestion
/// Memory tables in `information_schema`.
const MEMORY_TABLES: &[&str] = &[ENGINES, COLUMN_PRIVILEGES, COLUMN_STATISTICS];
```